main/src/com/google/refine/io/ProjectMetadataUtilities.java

```diff
@@ -33,12 +33,7 @@ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
 
 package com.google.refine.io;
 
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.FileReader;
-import java.io.IOException;
-import java.io.OutputStreamWriter;
-import java.io.Writer;
+import java.io.*;
```

Comment:
i would rather leave it as full package.   cc @antoine2711

Reply:
Oh yes, thank you. Didn't realise that slipped through 😄